datadog_checks_dev/datadog_checks/dev/tooling/specs/docs/core.py

```diff
@@ -43,7 +43,9 @@ def normalize_links(self):
             sections = deque(enumerate(file['sections'], 1))
             while sections:
                 sidx, section = sections.popleft()
+                section['prepend_text'] = self._normalize(section['prepend_text'], fidx, sidx)
                 section['description'] = self._normalize(section['description'], fidx, sidx)
+                section['append_text'] = self._normalize(section['append_text'], fidx, sidx)
```

Comment:
Will the link normalization work when separated like this?  Would be helpful to add a test that includes a link in prepend or append and verify it gets processed with the rest of the description correctly.

Reply:
just added tests for this, does it make sense?